scio-examples/src/test/scala/com/spotify/scio/examples/extra/AlgebirdSpec.scala

```diff
@@ -241,11 +241,13 @@ object AlgebirdSpec extends Properties("Algebird") {
 
   property("aggregator with BloomFilter") = forAll { xs: SColl[String] =>
     val width = BloomFilter.optimalWidth(1000, 0.01)
-    val numHashes = BloomFilter.optimalNumHashes(1000, width)
-    val m = BloomFilter(1000, 0.01)
-    val bf = xs.aggregate(BloomFilterAggregator(numHashes, width))
-    // BF should test positive for all members
-    xs.internal.forall(bf.contains(_).isTrue)
+    (width.isDefined) ==> {
```

Comment:
@ravwojdyla thoughts on this? In this version of Algebird `BloomFilter.optimalWidth` returns `Option[Int]` instead of `Int`, is it alright to just skip whenever it can't meet the required probability and returns `None`?